modules/agent-policy/scripts/create-update-script.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+####################################################################
+## This script safely creates or updates an agent policy.
+## The script takes eight arguments: PROJECT_ID, POLICY_ID,
+## DESCRIPTION, AGENT_RULES_JSON, GROUP_LABELS_JSON, OS_TYPES_JSON,
+## ZONES_JSON, and INSTANCES_JSON. This script is run
+## during `terraform apply`
+####################################################################
```

Comment:
I recommend adding `set -e` so errors will stop execution.

Reply:
added, thanks!

Reply:
Got it, thanks.